chat/src/Components/Message/Message.tsx

```diff
@@ -75,6 +75,19 @@ export const Message: FC<{
             }
           )}
         >
+          <span>
+            {badges?.length > 0 &&
```

Comment:
Since we are ensuring that badges will always be an array, either empty or with some images, no need for the conditional operator `?`.. In simple words, we know that badges are always an array.

Reply:
also key was missing there. i fixed that